libs/ui/src/lib/notification/notification.component.html

```diff
@@ -1,15 +1,26 @@
-<div class="cvi-notification cvi-notification--size-{{size}} cvi-notification--severity-{{severity}}" dataAttribute="notification">
+<div class="cvi-notification cvi-notification--size-{{size}} cvi-notification--severity-{{severity}}"
+     dataAttribute="notification">
   <button *ngIf="showCloseButton && size !== 'compact'"
           aria-label="Close"
           class="cvi-notification__close-button"
           (click)="close()"
           type="button">
-    <cvi-ng-icon name="close" class="cvi-notification__close-button-inner" svgClass="cvi-notification__close-button-icon" dataAttribute="close-icon" [height]="16"></cvi-ng-icon>
+    <cvi-ng-icon name="close" class="cvi-notification__close-button-inner"
+                 svgClass="cvi-notification__close-button-icon" dataAttribute="close-icon" [height]="16"></cvi-ng-icon>
+  </button>
+  <button *ngIf="showCloseButton && size === 'compact'"
```

Comment:
No need to have another copy of the close button in the template – the different styling of the "compact" close button can be simply done through CSS ancestor selector. See the other comment.